codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttEncoder.java

```diff
@@ -102,10 +111,11 @@ private static ByteBuf encodeConnectMessage(
         MqttConnectPayload payload = message.payload();
         MqttVersion mqttVersion = MqttVersion.fromProtocolNameAndLevel(variableHeader.name(),
                 (byte) variableHeader.version());
+        MqttCodecUtil.setMqttVersion(ctx, mqttVersion);
```

Comment:
Please update the class javadoc to indicate that we can also encode messages using MQTT version 5.0.